app/src/main/java/com/android/bookswap/ui/Addbook/AddToBook.kt

```diff
@@ -0,0 +1,206 @@
+package com.android.bookswap.ui.Addbook
+
+import android.widget.Toast
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.android.bookswap.model.DataBook
+import com.android.bookswap.model.Languages
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AddToBook(listToBooksView: listToBooksView) {
+  // State variables to store the values entered by the user
+  var title by remember { mutableStateOf("") }
+  var author by remember { mutableStateOf("") }
+  var description by remember { mutableStateOf("") }
+  var rating by remember { mutableStateOf("") }
+  var isbn by remember { mutableStateOf("") }
+  var photo by remember { mutableStateOf("") }
+  var language by remember { mutableStateOf("") }
+  // Getting the context for showing Toast messages
+  val context = LocalContext.current
+
+  // Scaffold to provide basic UI structure with a top app bar
+  Scaffold(
+      modifier = Modifier.testTag("addBookScreen"),
+      topBar = {
+        TopAppBar(
+            // Title of the screen
+            title = { Text("Add Your Book", modifier = Modifier.testTag("addBookTitle")) },
+            // Icon button for navigation (currently no action defined)
+            navigationIcon = {
+              IconButton(onClick = {}) {
```

Comment:
I know it is a little useless to mention, but I think it would be better to leave the button out if it has no implementation yet. It might let the person who does it have more freedom.